docs/Configuration.rst

```diff
@@ -5981,6 +6036,163 @@ data is stored in the database.
+      In fact this wallet is used to initialize and send rewards to users and spaces wallets.
+
+- The HTTP endpoint is used by *users and spaces wallets* using `Web3.js <https://web3js.readthedocs.io/>`__ (Browser side) to communicate with the blockchain to send transactions, unlike the *Admin wallet* which uses `Web3j <https://web3j.io/>`__ (Server side) to send transactions.
+
+// TODO ajouter un schéma avec la blockchain, la UI d'eXo Wallet et le server eXo derrière qui communiquent ensemble (A DEMANDER AU DESIGNER de l'aide !!!!)
+
+Blockchain transaction settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- When sending a transaction to Ethereum blockchain, some generic properties are required, especially *gas limit* and *gas price*.
+
+  - **Gas limit**
+
+    The *gas limit* is the maximum gas that can take a transaction using *Cauri* contract. It can be modified using the following property:
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.limit=150000
+
+  - **Gas price**
+
+    The *gas price* parameters will determine transaction fee of operations made using eXo Wallet addon. In the UI, the user has three choices:
+
+      -  Cheap transaction: this option will use the gas price configured by the following property (in WEI):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.cheapPrice=4000000000
+
+      -  Normal transaction: this option will use the gas price configured by the following property (in WEI):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.normalPrice=8000000000
+
+      -  Fast transaction: this option will use the gas price configured by the following property (in WEI):
+
+      .. code-block:: jproperties
+
+            exo.wallet.transaction.gas.fastPrice=15000000000
+
+
+* When a transaction is sent and wasn't mined for several days, it will be marked as failed in eXo Wallet internal database.
+  To configure the number of days waiting for transaction to get mined, please use the following property:
+
+    .. code-block:: jproperties
+
+          exo.wallet.transaction.pending.maxDays=3
+
+
+Wallet types
+~~~~~~~~~~~~
+
+* **Admin wallet**
+
+  The *Admin wallet* is created automatically during first startup of eXo Platform server.
+
+  In order to be able to send transactions on blockchain using `Web3j<https://web3j.io/>` in server side, the private key of the *Admin wallet* is stored in internal database.
+  The private key is encrypted using two passwords:
+
+     -  One password that is coming from a keystore file ( see :ref:`Updating password encryption key <UpdatePasswordEncryptionKey>` )
+
+     .. warning:: The generated keystore file under `gatein/conf/codec` has to be backed up as a data folder, because it have a key file that is used to decrypt stored wallets private keys. If it's lost, all wallets private keys will be lost and consequently, all funds would be lost as well.
```

Comment:
```suggestion
     .. warning:: The generated keystore file under `gatein/conf/codec` MUST be backed up as a data folder, because it contains a key file that is used to decrypt stored wallets private keys. If it's lost, all wallets private keys will be lost and consequently, all funds would be lost and unrecoverable.
```